Sources/IR/Emitter.swift

```diff
@@ -57,11 +57,14 @@ struct Emitter {
   private var loops = LoopIDs()
 
   /// For each block, the state of `frames` where the block was first entered.
-  private var stackOnEntry: [ Block.ID: Stack ] = [:]
+  private var stackOnEntry: [ Block.AbsoluteID: Stack ] = [:]
```

Comment:
```suggestion
  private var stackOnEntry: [Block.AbsoluteID: Stack] = [:]
```

I should have caught this one on Dave's PR.